docs/blockchain/contracts/nft-contract/nft-actions/stofrcfg.a.experimental.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'stofrcfg.a'
+order: 36
+
+---
+
+# stofrcfg.a
+
+Set global Uniq offer configuration
+
+## Behavior
+
+Set global configurations for all Uniq offers that will be made on either a Uniq or a Uniq factory.
+
+## Technical Behavior
+
+Only `ultra.nft.ft` account can set buy offer config
```

Comment:
Should be
Only `ultra.nft.ft` account can set Uniq offer configurations.

Reply:
updated both files